scala/Charts/src/main/scala/TaxiSQL.scala

```diff
@@ -0,0 +1,44 @@
+import java.io.File
+
+import com.databricks.connect.DatabricksSession
+import org.apache.spark.sql.SparkSession
+import org.jfree.chart.{ChartFactory, ChartFrame, ChartUtils, JFreeChart}
+import org.jfree.data.category.DefaultCategoryDataset
+
+object TaxiSQL {
+  def createSession(): SparkSession = {
+    DatabricksSession.builder.remote().getOrCreate()
+  }
+
+  def main(args: Array[String]): Unit = {
```

Comment:
Do you want to merge them together?

I was thinking it might be a bit better this way, because it keeps the file short and unintimidating for other people to copy-paste and start doing some changes on top.

Reply:
There is no standard way we have for examples, but seems preferable that there is one single entry point for all the charts to be generated. 

You can still keep them in separate files and just pass the session around.

Reply:
I do notice that this is done in other examples here, and I think it is more of a downside.

I think the most important part of example is the ease of being able to reuse it. The shorter example is and the less files are involved the more convenient.